content/en/docs/tasks/administer-cluster/kubeadm/kubeadm-upgrade.md

```diff
@@ -109,26 +110,26 @@ The upgrade workflow at high level is the following:
     [preflight] Running pre-flight checks.
     [upgrade] Making sure the cluster is healthy:
     [upgrade] Fetching available versions to upgrade to
-    [upgrade/versions] Cluster version: v1.15.2
-    [upgrade/versions] kubeadm version: v1.16.0
+    [upgrade/versions] Cluster version: v1.16.0
+    [upgrade/versions] kubeadm version: v1.17.0
 
     Components that must be upgraded manually after you have upgraded the control plane with 'kubeadm upgrade apply':
     COMPONENT   CURRENT       AVAILABLE
-    Kubelet     1 x v1.15.2   v1.16.0
+    Kubelet     1 x v1.16.0   v1.17.0
 
     Upgrade to the latest version in the v1.16 series:
 
     COMPONENT            CURRENT   AVAILABLE
-    API Server           v1.15.2   v1.16.0
-    Controller Manager   v1.15.2   v1.16.0
-    Scheduler            v1.15.2   v1.16.0
-    Kube Proxy           v1.15.2   v1.16.0
-    CoreDNS              1.3.1     1.6.2
-    Etcd                 3.3.10    3.3.15
+    API Server           v1.16.0   v1.17.0
+    Controller Manager   v1.16.0   v1.17.0
+    Scheduler            v1.16.0   v1.17.0
+    Kube Proxy           v1.16.0   v1.17.0
+    CoreDNS              1.6.2     1.6.2
```

Comment:
If I remember well we there will be a CoreDNS  bump this cycle.... or I'm wrong?

Reply:
this change is not yet in 1.17.0 i will update it.